dist_tables/ddl.rst

```diff
@@ -61,6 +61,16 @@ Each created shard is assigned a unique shard id and all its replicas have the s
 
 You are now ready to insert data into the distributed table and run queries on it. You can also learn more about the UDF used in this section in the :ref:`user_defined_functions` of our documentation.
 
+.. note::
```

Comment:
This note doesn't belong here. AFAIK, we now error out if you try to distribute a table which already has some data in it. So we shouldn't have any pre-existing data. Maybe we can add a question in the FAQ to cover how to move data from a PostgreSQL table into a Citus distributed table?